src/routes/Home/index.js

```diff
@@ -4,7 +4,9 @@ import Header from "../../components/Header";
 import WorkSection from './WorkSection';
 import SkillSection from "./SkillSection";
 import ContactSection from "./ContactSection";
-import { contact, skills, jobHistories, shortDescription } from '../../config/cv.json';
+import CV from '../../config/cv.json';
```

Comment:
Need to add a loader to allow the object destructing on JSON!

Since a JSON does have a default or export webpack complains about it